src/components/application_manager/src/hmi_capabilities_impl.cc

```diff
@@ -926,11 +926,18 @@ bool HMICapabilitiesImpl::load_capabilities_from_file() {
   }
 
   try {
-    Json::Reader reader_;
+    Json::CharReaderBuilder builder;
+    const std::unique_ptr<Json::CharReader> reader_(builder.newCharReader());
+    JSONCPP_STRING err;
     Json::Value root_json;
+    const int jsonLength = static_cast<int>(json_string.length());
 
-    bool result = reader_.parse(json_string, root_json, false);
+    bool result = reader_->parse(json_string.c_str(),
+                                 json_string.c_str() + jsonLength,
+                                 &root_json,
+                                 &err);
     if (!result) {
+      std::cout << "Json parce fails: " << err << std::endl;
```

Comment:
@ydementieiev parsing

Reply:
@AByzhynar done in [1345b2e](https://github.com/smartdevicelink/sdl_core/pull/3155/commits/1345b2e4f52415e896a5fa3d232aa391c69751bb)